bark/benchmark/tests/py_benchmark_result_tests.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) 2019 fortiss GmbH
+#
+# This software is released under the MIT License.
+# https://opensource.org/licenses/MIT
+
+import unittest
+import os
+import random
+
+try:
+    import debug_settings
+except:
+    pass
+
+
+from bark.benchmark.benchmark_result import BenchmarkResult
+
+def random_result_data(size):
+    columns = ['value1', 'value2', 'value3', 'value4']
+    results = [ {column : random.randint(0, 100) \
+             for column in columns}  for _ in range(0, size)]
+    return results
+
+def random_history_data(history_num, hist_size):
+    histories = {i : bytearray(os.urandom(hist_size)) \
+             for i in range(0, history_num)}
+    return histories
+
+class TestConfig:
+    def __init__(self, idx, conf_size):
+        self.bytes = bytearray(os.urandom(conf_size))
+        self.config_idx = idx
+    def __eq__(self, other):
+        return self.bytes == other.bytes and \
+          self.config_idx == other.config_idx
+def random_benchmark_conf_data(num_confs, conf_size):
+    confs = [ TestConfig(i, conf_size)for i in range(0, num_confs)]
+    return confs
+
+class DatabaseRunnerTests(unittest.TestCase):
+    def test_dump_and_load_results(self):
+        result_data = random_result_data(size = 10)
+        br = BenchmarkResult(result_dict=result_data)
+        br.dump("./results")
```

Comment:
do we not need to delete this dump after the tests have finished? Consider adding a cleanup function?